recipes-openxt/xenclient-initramfs-shared-libs/xenclient-initramfs-shared-libs_1.0.bb

```diff
@@ -1,15 +0,0 @@
-PR="r1"
-LICENSE = "GPLv2"
-LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
-
-DEPENDS = "virtual/libc openssl trousers"
```

Comment:
openssl and trousers still get pulled in through other RDEPENDS?  Or openssl gets pulled in and trousers wasn't needed since tpm-tools-sa are standalone?

Reply:
`trousers` is pulled through `RDEPENDS`.
`OE` adds runtime dependencies on common dynamic objects (probably relevant for `openssl`) https://www.yoctoproject.org/docs/3.1/overview-manual/overview-manual.html#automatically-added-runtime-dependencies.